libcxx/include/__functional/reference_wrapper.h

```diff
@@ -64,6 +67,62 @@ class _LIBCPP_TEMPLATE_VIS reference_wrapper : public __weak_result_type<_Tp> {
   {
     return std::__invoke(get(), std::forward<_ArgTypes>(__args)...);
   }
+
+#if _LIBCPP_STD_VER >= 26
+
+  // [refwrap.comparisons], comparisons
+
+  friend constexpr bool operator==(reference_wrapper __x, reference_wrapper __y)
```

Comment:
Here and the other friends.
```suggestion
_LIBCPP_HIDE_FROM_ABI  friend constexpr bool operator==(reference_wrapper __x, reference_wrapper __y)
```